pylib/mozhginfo/mozhginfo/pushlog_client.py

```diff
@@ -25,13 +25,23 @@
 LOG = logging.getLogger('pushlog_client')
 JSON_PUSHES = "%(repo_url)s/json-pushes"
 VALID_CACHE = {}
+VERSION = 2
 
 
 class PushlogError(Exception):
     pass
 
 
-def query_pushes_by_revision_range(repo_url, from_revision, to_revision, version=2, tipsonly=1):
+def exact_revision_from_push(pushes):
+    """accept a list of push object and return a list of revision"""
+    revisions = []
+    for push in pushes:
+        revisions = revisions + [changeset.node for changeset in push.changesets]
+    return revisions
+
+
+def query_pushes_by_revision_range(repo_url, from_revision, to_revision, version=VERSION,
+                                   tipsonly=True, string=False):
```

Comment:
Hmm, I just want to make it more simpler. How about something like "meta_string" or "revisions_only"?